README.md

```diff
@@ -56,7 +56,7 @@ The following directory structure and the files must be there to run
 │    └── temp <br>
 └── hyp_best_train_weights_final.h5 <br>
 
-Root directory where this structure is located should also be given in the maple_configuration file. 
+Root directory where this structure is located should also be given in the maple_configuration file.
```

Comment:
nit: can change maple_configuration file to [mpl_config.py](https://github.com/PermafrostDiscoveryGateway/MAPLE_v3/blob/main/mpl_config.py)